_ml-commons-plugin/api.md

```diff
@@ -494,6 +494,43 @@ GET /_plugins/_ml/profile
 }
 ```
 
+### Example: Return auto deploy and node information
+
+When the [auto redeploy]({{site.url}}{{site.baseurl}}/ml-commons-plugin/cluster-settings#enable-auto-redeploy) cluster setting is set to `true`, the profile API returns additional deployment information, including deployment time, retry count, and worker node IDs where the model is deployed.
```

Comment:
Sorry, I missed this change,  the auto redeploy setting set to `true` doesn't change the profile API.  And the response below is not the profile API response, it's get model API response. 

Suggest remove this part